handbook/engineering/leadership/index.md

```diff
@@ -46,15 +46,20 @@ We have a 1hr sync every Monday at 9:30am PT.
 
 ### Weekly updates
 
-Engineering managers send an update to [engineering-leadership-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-leadership-status) on either Monday or Tuesday each week (suggestion: send after your weekly team meeting). Anyone at the company can join the group to subscribe to these updates. The purpose of the update is to keep interested parties (including the VP of Engineering, and peer managers) informed of the progress that each team is making toward their documented goals.
+**Goal:** Increase awareness of each team's progress, tie progress to larger company goals, and communicate it across the organization.
+
+Product managers (or engineering managers for teams that don't have a PM) send an update to [engineering-leadership-status@sourcegraph.com](https://groups.google.com/a/sourcegraph.com/g/engineering-leadership-status) every Friday by 1PM [PT](https://time.is/PT), or at EOD Thursday if you don't work on Friday. Delegate to someone else on your team to send the update when you are on vacation.
```

Comment:
@nicksnyder does this mean that weekly update emails should now be sent by the PM instead of the EM? If that's the case I can add this section to our Product Management page too.

Reply:
I confused myself, see update here. Going to revert this to previous language.

https://github.com/sourcegraph/about/pull/2303/files#r556041112